examples/gae-autoinstrument-example/README.md

```diff
@@ -0,0 +1,63 @@
+Google AppEngine Standard with OTel Auto-instrumentation
+============================
+**This sample app is a Maven project and builds separately from the rest of the rest of the examples.**
```

Comment:
Why is it built using Maven? Gradle appears to be supported for AppEngine: https://cloud.google.com/appengine/docs/standard/java-gen2/using-gradle.